deploy/objects/KernelWrapper.js

```diff
@@ -0,0 +1,46 @@
+const ZepToken = artifacts.require('ZepToken');
+const Release = artifacts.require('Release');
+const ContractDirectory = artifacts.require('ContractDirectory');
+
+const log = require('../logger');
+
+export default class KernelWrapper {
+  constructor(kernel, owner) {
+    this.kernel = kernel;
+    this.owner = owner;
+  }
+
+  async zepToken() {
+    return ZepToken.at(await this.kernel.token());
+  }
+
+  async mintZepTokens(to, amount) {
+    log("\nMinting ZEP tokens for the developer...");
+    const zepToken = await this.zepToken();
+    log("  ZepToken:", zepToken.address);
+    const mintTx = await zepToken.mint(to, amount, { from: this.owner });
+    log("  Tokens minted: ", mintTx.tx)
+    return mintTx;
+  }
+
+  async registerRelease(contracts, developer) {
+    log(`\nDeploying a new release... `);
+    const release = await Release.new({ from: developer });
+    log(`  Deployed at ${release.address}`);
+    log(`  Adding contracts to release`)
+    await Promise.all(contracts.map(async function ([contractName, contractKlazz]) {
+      const implementation = await contractKlazz.new();
+      await release.setImplementation(contractName, implementation.address, { from: developer });
+      log(`    ${contractName} implementation: `, implementation.address);
+    }));
```

Comment:
nit: can we use objects here? IMO it will look better having sth like:
```js
contracts.map(async (contract) => {
  const implementation = await contract.klazz.new();
  await release.setImplementation(contract.name, implementation.address, { from: developer });
})
```

Reply:
Most likely these objects will die before the end of the week, as they'll be absorbed by another repo with deployment scripts and tools